climetlab/core/ipython.py

```diff
@@ -42,7 +53,12 @@ def guess_which_ipython():
     if ipython_active.ipython_dir == "/deepnote-config/ipython":
         return ("deepnote", None)
 
-    if ipython_active.ipython_dir == "/home/jovyan/.ipython":
+    # if ipython_active.ipython_dir == "/home/jovyan/.ipython":
+    #     return ("jupyter-lab", None)
+
+    # modify by longtsing
```

Comment:
author names are in the git commit, no need to add it in the code.

Reply:
ok,please delete it and merge into the master